src/resources/js/tickets-tables.js

```diff
@@ -50,19 +50,18 @@
 
 	function tribe_toggle_sortable() {
 		if ( window.matchMedia( '( min-width: 786px )' ).matches ) {
-			if ( ! $( $table ).hasClass( 'ui-sortable' ) ) {
```

Comment:
`$table` should already be a jQuery object, not sure why we need to wrap it in `$()`?

Reply:
We don't, that is why I removed it.

Reply:
Ah, ignore me - I saw that as adding it... 🤦‍♂️